trailblazer/server/api.py

```diff
@@ -80,6 +82,21 @@ def get_analysis(analysis_id):
         return jsonify(error=str(error)), HTTPStatus.NOT_FOUND
 
 
+@blueprint.route("/analyses/<int:analysis_id>/jobs", methods=["POST"])
+def add_job(analysis_id):
```

Comment:
I'll skip the return type hint because it is awful and the endpoint is not called as a function so there is no point in providing a type hint for the return